x-pack/plugins/lens/public/metric_visualization/visualization.tsx

```diff
@@ -89,14 +89,14 @@ const toExpression = (
 
   const labelFont = fontSizes[state?.size || 'xl'];
   const labelToMetricFontSizeMap: Record<string, number> = {
-    xs: fontSizes.s.size,
-    s: fontSizes.m.size,
-    m: fontSizes.l.size,
-    l: fontSizes.xl.size,
-    xl: fontSizes.xxl.size,
-    xxl: fontSizes.xxl.size * 1.2,
+    xs: fontSizes.xs.size * 2,
```

Comment:
As mentioned offline, only the xs mapping should use the old font sizes, the others should use the new one.

Reply:
Yep, my updates changed only xs mapping